plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -1246,57 +1246,27 @@ def build_transaction(  # pylint: disable=too-many-arguments
         """
         method = getattr(contract_instance.functions, method_name)
         tx = method(**method_args)
-        tx = self._build_transaction(
-            tx,
-            tx_args["sender_address"],
-            tx_args["eth_value"],
-            tx_args["gas"],
-            tx_args["gas_price"],
-            tx_args["max_fee_per_gas"],
-            tx_args["max_priority_fee_per_gas"],
-        )
-        return tx
 
-    def _build_transaction(  # pylint: disable=too-many-arguments
-        self,
-        tx: Any,
-        sender_address: str,
-        eth_value: int = 0,
-        gas: Optional[int] = None,
-        gas_price: Optional[int] = None,
-        max_fee_per_gas: Optional[int] = None,
-        max_priority_fee_per_gas: Optional[int] = None,
-    ) -> Optional[JSONLike]:
-        """Adds parameters to a transaction
-
-        :param tx: the transaction
-        :param sender_address: the sender's address
-        :param eth_value: the value of ETH to move
-        :param gas: the transaction gas
-        :param gas_price: the transaction gas price
-        :param max_fee_per_gas: the transaction max_fee_per_gas
-        :param max_priority_fee_per_gas: the transaction max_priority_fee_per_gas
-        :return: the final transaction
-        """
-        nonce = self.api.eth.get_transaction_count(sender_address)
+        nonce = self.api.eth.get_transaction_count(tx_args["sender_address"])
         tx_params = {
             "nonce": nonce,
-            "value": eth_value,
+            "value": tx_args["eth_value"] if "eth_value" in tx_args else 0,
```

Comment:
Generally speaking I'd change this to `value` here as this is also the web3 terminology. And over here we are closer to that then to our other repo's semantics.